src/librustdoc/html/format.rs

```diff
@@ -259,6 +259,12 @@ impl fmt::Display for clean::Lifetime {
     }
 }
 
+impl fmt::Display for clean::Constant {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}: {}", self.expr, self.type_)
```

Comment:
Make sure this honors the `f.alternate` flag - it renders plain-text instead of HTML links and is used to calculate line length for line wrapping. The easiest way is to use plain `fmt` calls with a `write_str` between them, and let the inner Display impls process the flag themselves.